### DISCUSSION TITLE
WLANpi NEO3?

### DISCUSSION THREAD

**Discussion by Devinator** [WLAN-Pi/feedback#20] [General Inquiries] [answered]
https://www.friendlyelec.com/index.php?route=product/product&product_id=279 << Any idea whether this hardware will work with the 2.x wlanpi image?

thanks!

Devin

**Comment by jiribrejcha** [comment, marked as answer]
It won’t work with the 2.x image, Devin. We built an experimental image for NEO3, and it worked in principle, but it is now outdated, untested and not supported.

**Comment by jiribrejcha** [comment]
Also, Armbian distribution stopped supporting NEO3:
https://www.armbian.com/nanopineo3/

WLAN Pi Community Edition running 3.x might be the right for you - Raspberry Pi 4, or MCU board with CM4.

**Comment by Devinator** [comment]
I really appreciate the help Jiri! Thanks for the fast and detailed answers!

**Reply by onlyfi** [reply]
Is there any other alternative for Neo2? Would the code work on Neo 1.4 kit? I will be buying a WLAN pi pro when it is available. However, at the moment I need a sensor to play with. Thanks in advance.

**Reply by jiribrejcha** [reply]
Raspberry Pi 4 running the latest 3.x release would be what I would recommend. It runs the latest code that we also use on the Pro, and hopefully you might be able to purchase an RPi4 locally. I appreciate the supply chain situation.

To boot 2.x image on NEO1, you would need an Armbian image compatible with the NEO1 and that's not available and it would take quite some effort to build it, test it and make it available to you. If you are after a generic Armbian OS image, which you could then manually customise, you can [download it from Armbian](https://www.armbian.com/nanopi-neo/), but obviously that would be missing the WLAN Pi features.

**Reply by onlyfi** [reply]
Thanks Jiri. Much appreciated.
